cmd/collect.go

```
@@ -157,7 +160,7 @@ func init() {
 	CollectCmd.Flags().StringVarP(&idMap, "bmc-id-map", "m", "", "Set the BMC ID mapping from raw json data or use @<path> to specify a file path (json or yaml input)")
 
 	CollectCmd.MarkFlagsMutuallyExclusive("output-file", "output-dir")
-	CollectCmd.RegisterFlagCompletionFunc("format", completionFormatData)
+	_ = CollectCmd.RegisterFlagCompletionFunc("format", completionFormatData)
```

Any time we ignore an error, we should comment why we are ignoring it.